writing/freeform/shopping/qna.yaml

```diff
@@ -0,0 +1,18 @@
+---
+- answer: You need to buy salt, yeast and mozzarella.
+  question: I'm making pizza. I need flour, water, salt, yeast, tomato puree, mozzarella and basil. I have flour, water, tomato puree and basil. What do I need to buy?
+- answer: You need to buy salt.
```

Comment:
Shouldn't this be flour, if you already have water and salt?

Reply:
Resolved now